experimental/Schemes/RationalMap.jl

```diff
@@ -535,70 +548,102 @@ function pullback(phi::RationalMap, C::AbsAlgebraicCycle)
     sort!(all_U, lt=(x,y)->complexity(x)<complexity(y))
 
     # First try to get hold of the component via cheap realizations 
-    success = false
+    pullbacks = IdDict{AbsSpec, Ideal}()
     for U in all_U
       psi = cheap_realization(phi, U, V)
       U_sub = domain(psi)
-      J = pullback(psi)(saturated_ideal(I(V)))
+      pullbacks[U] = pullback(psi)(saturated_ideal(I(V)))
+    end
+      #J = pullback(psi)(saturated_ideal(I(V)))
+    function new_complexity(U::AbsSpec)
+      return sum(total_degree.(lifted_numerator(gens(pullbacks[U]))); init=0)
```

Comment:
```suggestion
      return sum(total_degree.(lifted_numerator(gens.(pullbacks[U]))); init=0)
```
from the looks of it. But I did not confirm with an example

Reply:
o.k. nevermind nonsense